aries-site/src/examples/foundation/background-colors/BasicLayoutExample.js

```diff
@@ -0,0 +1,100 @@
+import React, { useContext } from 'react';
+import {
+  Anchor,
+  Header,
+  Box,
+  Main,
+  ResponsiveContext,
+  Sidebar,
+  Text,
+} from 'grommet';
+import { AppIdentity } from '../../../components/content/AppIdentity';
+
+export const BasicLayoutExample = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <AppContainer>
+      <AppSidebar />
+      <Box flex overflow="auto">
+        <Box height={{ min: '100%' }}>
+          <Header
+            border={{ color: 'border-weak', side: 'bottom' }}
+            background="background"
+            fill="horizontal"
+            pad="small"
+          >
+            <AppIdentity title="App Name" brand="hpe" />
+          </Header>
+          <Main
+            fill={undefined}
+            flex={false}
+            pad="small"
+            // // remove height="xlarge" in prod, for demo purposes only
+            // height="xlarge"
+          >
+            <Box direction={size !== 'small' ? 'row' : 'column'} gap="medium">
+              <Box gap="medium" flex>
+                <Text>
+                  This is the main page content. It may include buttons, tables,
+                  forms, or any other kind of component.
+                </Text>
+                <Text>
+                  For more info on building page layouts, see our{' '}
+                  <Anchor label="page layouts guidance" />.
+                </Text>
```

Comment:
```suggestion
                <Paragraph margin="none">
                  This is the main page content. It may include buttons, tables,
                  forms, or any other kind of component.
                </Paragraph>
                <Paragraph>
                  For more info on building page layouts, see our{' '}
                  <Anchor label="page layouts guidance" />.
                </Paragraph>
```